src/mixins/search.js

```diff
@@ -19,7 +19,6 @@ export const Search = () => ({
       a.fetchResults()
     },
     fetchResults: (s,a,d) => {
-      (s.searchString.length || s.searchResults.length === 0) &&
```

Comment:
This was here to make sure we only searched when there was a search term or there were no current search results. I guess you have tested these scenarios. If all good then :shipit:

Reply:
Yeah it still only searched on page load, and when you reach the bottom of the page without it.